river/tree/hoeffding_adaptive_tree_regressor.py

```diff
@@ -45,6 +45,21 @@ class HoeffdingAdaptiveTreeRegressor(HoeffdingTreeRegressor):
     nominal_attributes
         List of Nominal attributes. If empty, then assume that all numeric attributes should
         be treated as continuous.
+    attribute_observer
```

Comment:
And here.

Reply:
Updated in commit [f1eab02](https://github.com/creme-ml/creme/pull/379/commits/f1eab0274d21fedd3f599f8597e6a2f0230cbe26).